src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/exceptions/OrderNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package com.wojciechbarwinski.demo.epic_board_games_shop.exceptions;
+
+public class OrderNotFoundException extends ApplicationException{
+
+
+    public OrderNotFoundException(String message) {
```

Comment:
Ten message tutaj spokojnie mógłby być intem/Integerem o nazwie orderId

Reply:
nie do końca. Zakładam że później będzie można szukać po innych wartościach, np. nazwie. String jest uniwersalny po prostu

Reply:
To wtedy dodamy drugi konstruktor ;) ale pobieranie po id jest na tyle typowym przypadkiem, że walnąłbym konstruktor z intem orderId